### PR TITLE
app-text/sigil: bump v1.7.0

### PR DESCRIPTION
**Pull request by arthurzam**
@mgorny A small version bump to my package

bump the version, and drop old stable version (1.5.0 remains as stable)
Version 1.7.0 works quite well, so I think we can push it to out users.

Note to myself: upstream might change python executable argument handling in CMake, so I might need to fix the passing of out selected single python.

### CODE REVIEW

File: app-text/sigil/sigil-1.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/regex[${PYTHON_USEDEP}]') )
+	system-mathjax? ( dev-libs/mathjax )
+"
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
```
Comment (ionenwks) [inline app-text/sigil/sigil-1.7.0.ebuild]: `${PYTHON_DEPS}` should be in BDEPEND too, don't know if build system needs it but the eclass runs python at build time and expects you to handle adding deps.

File: app-text/sigil/sigil-1.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		dev-python/html5lib[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/regex[${PYTHON_USEDEP}]') )
+	system-mathjax? ( dev-libs/mathjax )
+"
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
```
Comment (ionenwks) [inline app-text/sigil/sigil-1.7.0.ebuild]: Unimportant but can technically skip pkgconfig with cmake, it's a RDEPEND of cmake and not something you need to worry about (unless you are calling pkg-config yourself, like in the ebuild or so -- just to be proper given it wouldn't be cmake using it).

You'd need this when it's a ./configure using it though.

**Review by ionenwks — COMMENTED** [review]
Seems to run fine, but maybe time for some small ebuild improvements while here?

**Comment by arthurzam** [comment]
> Seems to run fine, but maybe time for some small ebuild improvements while here?

Thanks for the review - it was indeed time to do major cleanup.

File: app-text/sigil/sigil-1.7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+"
+
+S="${WORKDIR}/Sigil-${PV}"
```
[the remark this answers is not in the document]
Comment (arthurzam) [reply app-text/sigil/sigil-1.7.0.ebuild]: Thanks for this trick - quite useful!

**Review by ionenwks — APPROVED** [review]
Thanks, looks great.

**Pull request closed without merging**